### PR TITLE
By Paritosh Sanadhya(Member of Team Swarm Robotics)

### PR DESCRIPTION
**Pull request by Pasa1912**

### CODE REVIEW

**Comment by annushree21** [comment]
Few changes are needed.
1. Put Report and poster into single folder.
2. Team names and mentors name should be hyperlinked to their git profile.
3. Add at least one photo each in mechanical aspect and electronic aspect of readme file.

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@Pasa1912 Changes are required!
1. Poster and Report have to be present inside a folder named "Poster and Report".
2. Poster has to be in .jpg format. 
3. Poster is not up to the mark. Make sure that you have "Models and Robotics Section", "Srishti 2020", Project Abstract, a picture related to your project in your poster.
4. In Report, maintain uniformity among headings and sub-headings.
5. In Report, Applications has to be written after Cost Structure. Also, add Limitations and Reference at the end.
6. "Shapes" should not be present in the root directory. It can be placed inside Images and Videos/Images. But make sure that shapes are in .jpg format, not .ppt format.
7. CAD Images have to be present inside Mechanical Design folder, not inside Images and Videos/Images.
8. In README.md, Mechanical Aspect, and Electronics Aspect is not about components used in mechanical and electronics design. You should explain mechanical and electronics features of your project under that sub-heading in short and crisp way. You should have separate sub-heading named "Components Used" to list out your components. 
9. In README.md, hyperlink Team Members and Mentors names with their GitHub profile link.
10. In README.md, justify the text using HTML tags.

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@Pasa1912 Also try not to use images obtained from internet!

**Comment by Pasa1912** [comment]
> @Pasa1912 Also try not to use images obtained from internet!

@SanjeevKrishnan  Sir, we don't have enough images of our project.

**Comment by SanjeevKrishnan** [comment]
> @SanjeevKrishnan Sir, we don't have enough images of our project.

@Pasa1912 You can use full view CAD Images

**Comment by agshubhs** [comment]
@Pasa1912 
1. Write content of readme normal not in italics.
2. Do not use any internet photo.
3. Write team members and mentors as ordered list not unorders list.
4. Add solid works jped in readme also.
5. For video you can upload bot recognition video using shapes and all.
6. Flow chart is missing.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,101 @@
...
+
+## Electronics Aspect of the Design
+In each of the bot, the microcontroller used is ESP8266(NodeMCU). For angle detection fron the initial position for a particular bot, a gyroscope sensor(particularly
+MPU-92/65 is used. For controlling the motors, a motor driver(particularly L298) is used in each bot.
+<p align="center">
+  <img width="600" height="400" src="/Mechanical Design/Solidworks_design/CAD_and_circuit_images/Img5.png">
+</p>
+  Initially each bot is set to a particular same direction, but when the user calls for a shape using a user friendy interface, each bot is assigned its final coordinates in the field of view of the overhead camera. Thus each bot reaches to its final position one by one. And the final shape is achieved.
+
+## Cost Structure of the Components Used
```
Comment (SanjeevKrishnan) [inline README.md]: Make this "Cost Structure" instead of "Cost Structure of the Components Used"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,101 @@
...
+| Nut & Bolt (Medium)      |     16   |          5    |         80 |
+| Nut & Bolt (Long)        |     16   |         10    |        160 |
+| L - Brackets             |      8   |         15    |        120 |
+| Normal Wheels            |      8   |         30    |        240 |
+| Castor Wheels            |      4   |         25    |        100 |
+| PlyWood                  |      1   |        280    |        280 |
+| Miscellaneous            |      -   |          -    |        600 |
+| Total                    |          |               | Rs.18000/- |
+
+## Working
```
Comment (SanjeevKrishnan) [inline README.md]: Move Working above Mechanical Aspect

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@Pasa1912 Very minor Changes
1. Use `<p align="justify">Content<\p>` to justify text in README.md where Content is the text you want to justify.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,100 @@
 # Swarm_Robotics
-Srishti 2020
+## Srishti 2020
+<p align ="justify">
```
Comment (SanjeevKrishnan) [inline README.md]: @Pasa1912 Remove this `<p>` tag. Use this tag separately for individual paragraphs which you want to justify

**Review by SanjeevKrishnan — APPROVED** [review]
@Pasa1912 Great Work!
@Shubhanshu07 I suggest for merge

**Comment by agshubhs** [comment]
@Pasa1912 Looks good to  me

**Pull request merged**